src/wasm-interpreter.h

```diff
@@ -2425,6 +2425,7 @@ template<typename GlobalManager, typename SubType> class ModuleInstanceBase {
   void initializeTableContents() {
     ModuleUtils::iterActiveElementSegments(wasm, [&](ElementSegment* segment) {
       Function dummyFunc;
+      dummyFunc.type = Signature(Type::none, Type::none);
```

Comment:
Hmm, Signature having a default value but HeapType not does mean we may have bugs in places we missed adding such a thing... but seems not likely to be a problem.

Reply:
It's probably worth giving Function a default HeapType corresponding to the empty signature. I'll do that.